chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -91,7 +91,8 @@ def self.bridge?(nic)
   def self.ovs_bridge?(nic)
     nic.kind_of?(::Nic::OvsBridge) or
       # If no ovs tools are installed there's no ovs switch
-      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1")
+      if ::Kernel.system("which ovs-vsctl > /dev/null 2>&1 && " \
```

Comment:
Is `which` still installed per default on SLE 12? If I remember correctly you should use `hash` instead of `which` as you are touching that code anyway.